app/api/routers/attributes.py

```diff
@@ -30,6 +39,9 @@ async def get_terms(
 
     if data_element_URI == DataElementURI.assessment:
         term_labels_path = request.app.state.cogatlas_term_lookup_path
+    else:
```

Comment:
is there are reason you add the `else` here?

Reply:
Hm, I think you're right that it's not actually needed!